TVC_MVP/CountyPresenter.swift

```diff
@@ -84,6 +91,23 @@ class CountyPresenter: CountyPresenterProtocol {
         }
     }
 
+    private func generateAlertMessage(county: County) -> String {
+        var newbornString: String = ""
```

Comment:
When creating a string instance you don't need to specify is a `String`. The scope of this variable is not right as we just need it **only** when we have `extras`. 

Either way, do you think that we really need the instances?

When building a string that can mutate based on different properties, and there are different paths with `ifs`, we would usually have the base string, and then **append** to it, having a single `return` at the end of the method.